Add option to disable TUI mode spacing

Octelly opened the pull request.

Octelly committed:
added quotation wrapping around layout name

Octelly committed:
Merge branch 'echuraev:master' into master

Octelly committed:
Added option to disable TUI spacing

Octelly committed:
Merge branch 'master' of github.com:Johnystar/keyboard_layout into HEAD

echuraev commented on kbdcfg.lua:
```diff
@@ -130,6 +134,7 @@ local function factory(args)
     kbdcfg.current               = args.current or kbdcfg.default_layout_index
     kbdcfg.menu                  = nil
     kbdcfg.type                  = args.type or "tui"
+    kbdcfg.no_tui_margin         = args.no_tui_margin or false
```
I would prefer if we will have option `tui_space_symbol` or something like that. And by default it should be equal to " ". In this case, user will be able to customize it. This option will be more flexible as `no_tui_margin`.

echuraev reviewed (changes requested):
Thank you for your PR. Could you please also update the README.md file and describe the new option?

Octelly commented:
> Thank you for your PR. Could you please also update the README.md file and describe the new option?

thank you for the feedback, I'm a bit busy at the moment, but I'll try to get to it as soon as possible

Octelly committed:
Changed TUI spacing toggle to customisabile wrap

Octelly commented:
I've updated my fork and have taken the liberty of making it my way once again. I've made it so that the user has full control over the wrapping around the widget. This allows me to easily do this (for example):
```lua
-- keyboard layouts
local kbdcfg = keyboard_layout.kbdcfg({
    type = "tui",
    tui_wrap = "",
    tui_wrap_left = " "
})
```
![image](https://user-images.githubusercontent.com/29976879/171179521-96fb24c9-3a58-436a-a2da-e61cdde1ba79.png)

echuraev commented on kbdcfg.lua:
```diff
@@ -130,6 +130,9 @@ local function factory(args)
     kbdcfg.current               = args.current or kbdcfg.default_layout_index
     kbdcfg.menu                  = nil
     kbdcfg.type                  = args.type or "tui"
+    kbdcfg.tui_wrap              = args.tui_wrap or " "
```
I have some doubts about this variable. Do you think that we really need it? Probably it is enough to give only one option for the user (to specify symbol for left and right wrappings separately). Too many options can confuse. When the user wants to configure the wrapping, I don't think that it will be a problem to initialize two variables instead of one.

Octelly replied:
My thinking was that most users would probably just want to change ``tui_wrap`` and not both settings separately, but I suppose you're right.

echuraev reviewed (changes requested):
Thank you for your PR. It LGTM, but I have one question about `tui_wrap` variable. Also, could you please add the documentation for new variables to the README.md?

Octelly committed:
Reduced the amount of options to simplify widget configuration

Octelly committed:
Added an explanation of TUI wrap configuration

Octelly commented:
I removed the ``tui_wrap`` variable and kept the rest. Configuration on my side now looks like this:
```lua
-- keyboard layouts
local kbdcfg = keyboard_layout.kbdcfg({
    type = "tui",
    tui_wrap_right = "",
    tui_wrap_left = " "
})
```

I also added an explanation of how to use the wrapping feature to the readme.

echuraev reviewed (changes requested):
Thank you for your contribution. It looks awesome :) The last thing that I'd like to ask you. Could you please create folder `docs` and move your screenshot and also other gif files which are used in the README.md to this folder? I think it would help to keep the root of the project as clean as possible.

Octelly committed:
Moved README resources into docs/

echuraev reviewed (approved):
LGTM. Thank you for your contribution!
